### PR TITLE
Introduce inspection for embedded interface pointer (fixes #1374)

### LINKED ISSUE #1374: Should be an error: embedded type cannot be a pointer to interface
```go
package main

import "fmt"

func main() {
    c := Parent{&Child{}, &impl{}}
    c.Get(1,1,1)
}

type Parent struct {
    *Child
    *Interface // embedded type cannot be a pointer to interface
}
type Child struct { Unimplemented }
type Interface interface { Get(...int) }
type impl struct { }
func (r impl) Get(h ...int) { fmt.Println("HERE") }
type Unimplemented interface { Get(s string) }
```

### PR DESCRIPTION
**Pull request by dlsniper**

### CODE REVIEW

File: src/com/goide/inspections/GoEmbeddedInterfacePointerInspection.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.jetbrains.annotations.NotNull;
+
+public class GoEmbeddedInterfacePointerInspection extends GoInspectionBase {
+  @NotNull
+  @Override
+  protected GoVisitor buildGoVisitor(@NotNull final ProblemsHolder holder, @NotNull LocalInspectionToolSession session) {
+    return new GoVisitor() {
+      @Override
+      public void visitAnonymousFieldDefinition(@NotNull GoAnonymousFieldDefinition o) {
+        if (o.getMul() == null) return;
```
Comment (zolotov) [inline src/com/goide/inspections/GoEmbeddedInterfacePointerInspection.java]: please add test when mul == null

Comment (dlsniper) [inline src/com/goide/inspections/GoEmbeddedInterfacePointerInspection.java]: Done

**Comment by zolotov** [comment]
thanks!
